README.md

```diff
@@ -16,3 +16,18 @@ To run backend:
 `cd backend && npm install -d`  
 then  
 `npm run dev`
+
+### How to connect Google Identity API
```

Comment:
Dla wygody deweloperów myślę, że można zrobić jakieś dedykowane konto google dla AlgoDebuga tam wyklikać co trzeba i po prostu się udostępni reszcie credentiale. Wtedy tą informację bardziej bym widział na wiki niż w README projektu.

Reply:
mamy jakieś wiki?

Reply:
Na githubie jest zakładka wiki, tam można dodawać swoje strony. Tylko dajcie znać, czy macie uprawnienia do edycji wiki. (a githubie nie korzystałem jeszcze z wiki, ale na gitlabie spoko to działało)